CTM-2140: Asset data points and linking documentation (#3478)
SoftwareAG/c8y-docs: content/dtm/asset-hierarchy-bundle/managing-asset-datapoints.md

3. Click the **Change source** option in the context menu of the linked data point.
4. In the **Change source** dialog, you can select a new source device or modify the fragment and series of the linked data point.
5. After making your changes, click **Save** to apply the changes.

### To Delete Linked Data Points

1. Navigate to the **Assets** page and select the asset for which you want to delete linked data points.
2. In the **Data points** section, find the linked data point you wish to delete
3. Hover over the linked data point and click the delete icon <i class="dlt-c8y-icon-editing-trash text-danger icon-16"></i> that appears at the right.
4. In the confirmation dialog, select **Delete** to delete the linked data point.

### To Unlink Data Points

1. Navigate to the **Assets** page and select the asset for which you want to unlink data points.
2. In the **Data points** section, find the linked data point you wish to unlink.
3. Click the **Unlink source** option in the context menu of the linked data point.
4. In the confirmation dialog, select **Unassign** to remove the link between the asset and the source device for that data point.

{{< c8y-admon-info>}}

> Unlinking a data point does not delete the data point itself; it simply removes the association between the asset and the source device. The data point will still exist in the system but will no longer be linked to the asset. The state of the data point will change to *Incomplete* after unlinking, indicating that it is no longer associated with a source device.

Reviewer: ```suggestion
Unlinking a data point does not delete the data point itself; it simply removes the association between the asset and the source device. The data point will still exist as a measurement of the source device but will no longer be linked to the asset. The state of the data point will change to *Incomplete* after unlinking, indicating that it is no longer associated with a source device.
```
